### PR TITLE
Fix invite link

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

File: go/libkb/invite.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func (i Invitation) Link() string {
 	if i.Throttled {
 		return ""
 	}
-	return path.Join(CanonicalHost, "inv", i.ShortCode[0:10])
+	return path.Join(CanonicalHost, "inv", i.ID[0:10])
```
Comment (maxtaco) [inline go/libkb/invite.go]: does it work on the server to only see the first 10 letters of the code?

Comment (patrickxb) [inline go/libkb/invite.go]: yes

Comment (patrickxb) [inline go/libkb/invite.go]: (that's how @malgorithms specified it)

**Comment by maxtaco** [comment]
👍

**Pull request merged**